samples/drivers/uart/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  name: UART driver sample
+tests:
+  sample.drivers.uart:
+    tags: serial uart
+    filter: CONFIG_SERIAL and
+            CONFIG_UART_INTERRUPT_DRIVEN and
+            dt_chosen_enabled("zephyr,shell-uart")
+    harness: keyboard
+    harness_config:
+        type: multi_line
+        ordered: true
+        regex:
+            - "Hello! I'm your echo bot."
+            - "Tell me something and press enter:"
+            - "Echo: (.*)"
```

Comment:
This part is not needed:
`harness: keyboard` requires user interaction, which never happens in CI.
(also, twister simply fails if uart console is busy with another program)

Reply:
Ok, I admit I was not sure if it made sense. Removed it.